README.md

```diff
@@ -1,4 +1,4 @@
-# Rust port of pandoc-types [![Build Status](https://travis-ci.org/elliottslaughter/rust-pandoc-types.svg?branch=master)](https://travis-ci.org/elliottslaughter/rust-pandoc-types)
+# Rust port of pandoc-types ![Build status](https://github.com/nokome/rust-pandoc-types/actions/workflows/test.yml/badge.svg)
```

Comment:
This would need to change to point to the original repo.

Reply:
Fixed.